### PR TITLE
Add navbar to the new Admin Panel

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | bootstrap-admin-panel (2.0)
| Bug fix?        | no
| New feature?    | 
| BC breaks?      | 
| Deprecations?   | 
| Related tickets | based on #15341
| License         | MIT

CleanShot 2023-09-22 at 11 54 24@2x

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{% set data = resources.data %}
+{% set event_name = metadata.applicationName ~ '.admin.' ~ 'index.body' %}
+{% set resource_event_name = metadata.applicationName ~ '.admin.' ~ metadata.name ~ '.index.body' %}
+
+{% set header = configuration.vars.header|default(metadata.applicationName~'.ui.'~metadata.pluralName) %}
+
+{% block title %}{{ header|trans }} {{ parent() }}{% endblock %}
+
+{% block body %}
+    {{ sylius_template_event(event_name, { event_prefix: event_name, resource_event_prefix: resource_event_name }) }}
```
Comment (loic425) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/index.html.twig:13]: You have an issue here, cause you remove the fallback for 'sylius.admin.index.body' when you have'nt define 'app.admin.index.body' (for custom entities).

metadata.applicationName could be different than "sylius".

Comment (loic425) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Shared/Crud/index.html.twig:13]:
```suggestion
    {{ sylius_template_event([event_name, 'sylius.admin.index.body'], { event_prefix: event_name, resource_event_prefix: resource_event_name }) }}
```

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/twig_component.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ This file is part of the Sylius package.
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/twig_component.xml:4]: Docblock contains extra `~` symbols at the start of each line

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/twig_component.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  ~
+  ~ (c) Sylius Sp. z o.o.
+  ~
+  ~ For the full copyright and license information, please view the LICENSE
+  ~ file that was distributed with this source code.
+  -->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Sylius\Bundle\AdminBundle\TwigComponent\Shared\Navbar\NotificationsComponent">
```
Comment (diimpp) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/twig_component.xml:14]: If I will want to override this service as end-user, my services definition will look this this
```yaml
services:
    Sylius\Bundle\AdminBundle\TwigComponent\Shared\Navbar\NotificationsComponent:
        class: App\TwigComponent\Admin\Shared\Navbar\NotificationsComponent
```

What's the best practice here? To set a non-class service `id` or something else?

**Pull request merged**